src/waveform/renderers/waveformwidgetrenderer.h

```diff
@@ -162,6 +162,9 @@ class WaveformWidgetRenderer {
     DISALLOW_COPY_AND_ASSIGN(WaveformWidgetRenderer);
     friend class WaveformWidgetFactory;
     QMap<WaveformMarkPointer, int> m_markPositions;
+    // draw play position indicator triangles
+    void drawPlayPosmarker(QPainter* painter);
+    void drawTriangle(QPainter* painter, QBrush fillColor, qreal x0, qreal y0, qreal x1, qreal y1, qreal x2, qreal y2);
```

Comment:
Actually `int x1` etc. would suffice. Do we need `qreal` here in case the GUI is scaled?